CauldronMods/Controller/Environments/StSimeonsCatacombs/Cards/AqueductsCardController.cs

```diff
@@ -16,13 +17,12 @@ public AqueductsCardController(Card card, TurnTakerController turnTakerControlle
 
         public override void AddTriggers()
         {
-            //At the end of the environment turn, each target regains 1 HP
-            base.AddEndOfTurnTrigger((TurnTaker tt) => tt == base.TurnTaker, (PhaseChangeAction pca) => this.AllTargetsGainHP(), TriggerType.GainHP);
-
+            //At the start of the villain turn, each target regains 1 HP
+            AddStartOfTurnTrigger((TurnTaker tt) => FindVillainTurnTakerControllers(true).Contains(FindTurnTakerController(tt)), AllTargetsGainHP, TriggerType.GainHP);
```

Comment:
Can this just be tt => tt.isVillain

Reply:
Maybe? I copied this from Riding the Currents. I think that would work just fine